packages/block-library/src/index.js

```diff
@@ -217,7 +228,7 @@ export const __experimentalRegisterExperimentalCoreBlocks =
 								postExcerpt,
 								postFeaturedImage,
 								postTags,
-						  ]
+						  ].filter( Boolean )
```

Comment:
@Addison-Stavlo that's to discard non-truthy elements from the array through the `Boolean` function (basically a "cast to bool").
E.g.
```js
blocksArray.filter( item => Boolean( item ) );
```

Reply:
Ohh gotcha, that makes sense.  Thanks!  So it will get rid of the `null` value.  For whatever reason I was assuming it was filtering out `typeof === 'boolean'` 🤦‍♀️  😆